### PR TITLE
Photo grab 

### PR DESCRIPTION
**Pull request by New-dev0**
https://github.com/code-rgb/USERGE-X/pull/92

### CODE REVIEW

File: plugins/profile.py [hunk 1]
```diff
@@ -182,8 +182,7 @@ async def set_profile_picture(message: Message):
             "-lname": "Print only last name",
             "-flname": "Print full name",
             "-bio": "Print bio",
-            "-uname": "Print username",
```
Comment (code-rgb) [inline plugins/profile.py:185]: Why you deleted "-uname": "Print username",

File: plugins/profile.py [hunk 2]
```diff
@@ -391,7 +381,32 @@ async def clone_(message: Message):
...
+        "header": "upload the photo of replied person or chat",
+        "usage": "upload the photo of replied person or chat",
+        "examples": ".poto",
+    },
+    allow_channels=False,
+)
+async def photograb(message: Message):
+    if message.reply_to_message and message.reply_to_message.from_users.photo:
+        getid = message.reply_to_message.from_user.photo.big_file_id
+        getphoto = await userge.download_media(getid)
```
Comment (code-rgb) [inline plugins/profile.py]: change userge to message.client (can be done both by bot and userbot)

File: plugins/profile.py [hunk 3]
```diff
@@ -391,7 +381,32 @@ async def clone_(message: Message):
...
+        await userge.send_photo(message.chat.id,photo=getphoto)
+        os.remove(getphoto)
+    elif message.chat.photo and not message.reply_to_message:
+        phid = message.chat.photo.big_file_id
+        ppo = await userge.download_media(phid)
+        await userge.send_photo(message.chat.id,photo=ppo
```
Comment (code-rgb) [inline plugins/profile.py]: missing ')' at photo=ppo

File: plugins/profile.py [hunk 4]
```diff
@@ -391,7 +381,32 @@ async def clone_(message: Message):
...
+        os.remove(ppo)
+    else:
+        await message.edit_text("Didnt Found Anything !")
```
Comment (code-rgb) [inline plugins/profile.py]: message.edit_text() -> message.err()

**Review by code-rgb — CHANGES_REQUESTED** [review]

File: plugins/profile.py (hunk 1 above)
Comment (New-dev0) [reply plugins/profile.py:185]: Its still there 🤔

**Comment by New-dev0** [comment]
Remaining changed done

**Pull request merged**